### PR TITLE
Add tag to unit tests in Haiku and Wordle kata

### PR DESCRIPTION
**Pull request by ivalkshfoeif**
fix for issue #283

### CODE REVIEW

**Review by rachanavishwanath — APPROVED** [review]

**Comment by prathasirisha** [comment]
Hi @ivalkshfoeif - can you update your PR , you can do this by - 

git pull --rebase upstream master 
git push --force-with-lease

**Comment by ivalkshfoeif** [comment]
Hi @prathasirisha, I have updated my PR, please let me know if there is anything else I need to do.

File: haiku-kata/src/test/java/org/eclipse/collections/haikukata/TextProcessorECTest.java [hunk 1]
```diff
@@ -27,6 +27,7 @@
 public class TextProcessorECTest
 {
     @Test
+    @Tag("KATA")
```
Comment (prathasirisha) [inline haiku-kata/src/test/java/org/eclipse/collections/haikukata/TextProcessorECTest.java:30]: It seems the builds are failing as it is not able to find the class "Tag". You are missing import in this class. 
`import org.junit.jupiter.api.Tag;`

Same comment on the 4 files.

Comment (ivalkshfoeif) [reply haiku-kata/src/test/java/org/eclipse/collections/haikukata/TextProcessorECTest.java:30]: Thank you so much for your guidance @prathasirisha , I will update it.

**Comment by prathasirisha** [comment]
After addressing the comment above, could you squash all your commits to 1 commit?

**Comment by ivalkshfoeif** [comment]
> After addressing the comment above, could you squash all your commits to 1 commit?

**Review by prathasirisha — APPROVED** [review]

**Pull request merged**